gallery/src/components/NxList/examples/NxListMultiLine.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxList } from '@sonatype/react-shared-components';
+
+function NxListMultiLine() {
```

Comment:
It's debatable whether this example is necessary.  It isn't a variation of how to use the component props, but of how to use styling classes with it - which I think is fine to cover only in the `.nx-list` example page.

Reply:
> It's debatable whether this example is necessary. It isn't a variation of how to use the component props, but of how to use styling classes with it - which I think is fine to cover only in the `.nx-list` example page.

It has been removed.
abc411240192e9862f96343a1df37986c26891aa